dev/DynamicDependencyLifetimeManager/DynamicDependencyLifetimeManager.ProxyStub/DynamicDependencyLifetimeManager.ProxyStub.rc

```diff
@@ -3,4 +3,4 @@
 
 #include <winresrc.h>
 #define VERSIONINFO_FILENAME "DynamicDependencyLifetimeManager.ProxyStub.dll"
-#include "..\..\common\WindowsAppSDK.ver"
+#include "..\..\..\eng\common\VersionInfo\AssemblyInfo.ver"
```

Comment:
Don't do relative path'ing. Makes it a bear if you move a project or ever clone to a new one with different directory layout.

```
#include "AssemblyInfo.ver"
```

and add $(RepoRoot)\eng\common\VersionInfo to the AdditionalIncludeDirectories like here https://github.com/microsoft/WindowsAppSDK/pull/3296/files#diff-240878995896e256290816b018b60bd9f02c1cdb15e59940b14d0af40def0e2d lines 183-187